rfcs/InputUnion.md

```diff
@@ -255,7 +255,7 @@ In addition to containing Input types, member type may also contain Leaf types l
 |----|----|----|----|----|
 | 🚫 | 🚫 | ✅⚠️ | 🚫 | ✅ |
 
-Criteria score: 🥉
+Criteria score: 🥈
 
```

Comment:
I think that this might be higher importance than currently rated; it's certainly a popular request here: https://github.com/graphql/graphql-spec/issues/215

Reply:
I ranked it bronze is because it's not currently supported in object polymorphism so it'd create a brand new discontinuity between input and object polymorphism

Reply:
I believe this distinction is acceptable, and it's clear to me the functionality is desired. I don't think our existing output polymorphism should place unnecessary restraints on our input polymorphism solution - we may yet find a way to solve this with output polymorphism (e.g. with value boxing `{__typename: "Int", value: 3}`).